xr-active-switch.ts

```diff
@@ -0,0 +1,115 @@
+import {Component, Object3D, property} from '@wonderlandengine/api';
+
+export class XrActiveSwitch extends Component {
+    static TypeName = 'xr-active-switch';
+
+    /** Action to perform when XR session starts */
+    @property.enum(['Activate', 'Deactivate', 'Toggle', 'Keep'])
```

Comment:
I think enums on other components are usually all lowercase. Could you check if that's true and adjust if so?